src/codeu/chat/util/store/Store.java

```diff
@@ -74,6 +74,12 @@ public VALUE first(KEY key) {
   }
 
   @Override
+  public VALUE last(KEY key) {
```

Comment:
Did this method during testing and must've forgotten to remove this one, too. Everything would work with a first() since its only one key, hehe.